app/backend/wells/views.py

```diff
@@ -202,3 +202,23 @@ def get(self, request):
             return Response([])
         else:
             return super().get(request)
+
+
+class PreSignedDocumentKey(APIView):
+    """
+    Get a pre-signed document key to upload into an S3 compatible document store
+
+    post: obtain a URL that is pre-signed to allow client-side uploads
+    """
+
+    permission_classes = (DjangoModelPermissionsOrAnonReadOnly,)
```

Comment:
We're moving away from model permissions. You could leverage WellsEditPermissions right now if you wanted to.